HLTrigger/JetMET/plugins/HLTJetTimingProducer.h

```diff
@@ -0,0 +1,187 @@
+/** \class HLTJetTimingProducer
+ *
+ *  \brief  This produces timing and associated ecal cell information for calo jets 
+ *  \author Matthew Citron
+ *
+ */
+#ifndef HLTrigger_JetMET_plugins_HLTJetTimingProducer_h
+#define HLTrigger_JetMET_plugins_HLTJetTimingProducer_h
+
+// system include files
+#include <memory>
+
+// user include files
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
```

Comment:
If you want to keep separated .h and .cc files, this can be included in the implementation file